src/renderer/tvgText.cpp

```diff
@@ -71,6 +71,22 @@ Result Text::load(const std::string& path) noexcept
 }
 
 
+Result Text::load(const char* name, const char* data, uint32_t size, const string& mimeType, bool copy) noexcept
+{
+    if (!name) return Result::InvalidArguments;
+
+    //unload font
+    if (!data && size == 0) {
+        if (LoaderMgr::retrieve(name)) return Result::Success;
```

Comment:
It is not appropriate that the load() function returns Success indicating that the unload was successful when there is no data.
Does load(name, nullptr, 0, ...) mean unload?

Reply:
yes... it is unloading... I'm not very happy with such method, but please check [this](https://github.com/thorvg/thorvg/pull/2345#discussion_r1625910687) and the discussion around. 
I think we could adopt the unload api... so it expects some char* (which for now is string), being either a path or a name. but with name should come a mime type, otherwise if we have more font loaders, there may be a problem.